arobit/deploy/templates/forwarder-daemonset.yaml

```diff
@@ -101,34 +101,34 @@ spec:
             - name: flb-config
               mountPath: /forwarder/etc
               readOnly: true
-        {{- if .Values.forwarder.mdsd }}
+        {{- if .Values.forwarder.mdsd.enabled }}

```

Comment:
n.b. do you actually expect to have production use-cases for toggling all these `enabled` flags? If not, might be nice to simplify this template - can be pretty hard to grok what the end result looks like from this uber-flexible thing if we only ever use a tiny subset of the options